src/views/Pools/components/ClaimAction/index.tsx

```diff
@@ -11,19 +11,19 @@ const ClaimAction: React.FC = () => {
         <ActionContainer style={{ margin: '10px 0' }}>
       <ActionTitles>
         <Text bold textTransform='uppercase' color={theme.colors.MGG_accent2} fontSize='12px' pr='4px'>
-          test
+          MGG
         </Text>
         <Text bold textTransform='uppercase' color='textSubtle' fontSize='12px'>
           Earned
         </Text>
       </ActionTitles>
       <ActionContent>
         <div>
-          <Earned>123</Earned>
+          <Earned>0</Earned>
```

Comment:
padynamic pls. su earned value man lang ini

Reply:
This is noted, pa approved nalang kuys ta isabay ko nalang etong change request duman sa fix modal na branch if okay lang?

Reply:
This issue is for Banner ui update lang po sana, doon na ako sa modal fix na branch ma sabay kang dynamic data

Reply:
I'll include this change request here kasi eto yung fix for modals. Is it okay? @medardm  
pr link -> https://github.com/mgguild/gamevault-frontend/pull/19